functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py

```diff
@@ -0,0 +1,264 @@
+                           ("Positive values", 1, 200)])
+    def test001_create_account_with_different_options(self, type, factor, return_code):
+        """ OVC-000
+        *Test case for testing creating account wuth different options .*
+
+        **Test Scenario:**
+
+        #. Create account with passing negative values in the account's limitation, should fail.
+        #. Create account with certain limits, should succeed.
+        """
+        self.lg.info("Create account with passing %s values in the account's limitation." % type)
+        accounts_limitation = {"maxMemoryCapacity": random.randint(2, 1000) * factor,
+                               "maxVDiskCapacity": random.randint(2, 1000) * factor,
+                               "maxCPUCapacity": random.randint(2, 1000) * factor,
+                               "maxNetworkPeerTransfer": random.randint(2, 1000) * factor,
+                               "maxNumPublicIP": random.randint(2, 1000) * factor}
+        data, response = self.api.cloudbroker.accounts.create(username=self.user, **accounts_limitation)
+        self.assertEqual(response.status_code, return_code, "A resource limit should be a positive number or -1 (unlimited).")
+        if return_code == 200:
+            self.CLEANUP['accounts'].append(response.json())
+
+    def test002_create_account_with_non_existing_user(self):
+        """ OVC-000
+        *Test case for testing creating account with non existing user.*
+
+        **Test Scenario:**
+
+        #. Create account with non-exist user, should fail.
+
+        """
+        self.lg.info(" Create account with non-exist user, should fail.")
+        fake_user = self.utils.random_string()
+        data, response = self.api.cloudbroker.accounts.create(username=fake_user)
+        self.assertEqual(response.status_code, 400, "Email address is required for new users.")
+
+    @parameterized.expand([('R',200,403,403),
+                           ('RCX',200,403,200),
+                            ('ARCXDU',200,200,200)
+                            ])
+    def test003_add_user_to_account(self,accesstype,get_code,update_code,cloudspace_code):
+        """ OVC-000
+        *Test case for adding user to account with different accesstypes.*
+
+        **Test Scenario:**
+   
+        #. Create user [u1].
+        #. Create account[C1] for main user and get this account  with main user,should succeed.
+        #. Add user[U1] to [C1]with access[accesstype], should succeed.
+        #. If accesstype read [R], should succeed to get account[C1] with user[U1]
+                                 , should fail to update account[C1] name or create cloudspace on it.
+        #. If accesstype write [RCX], should succeed to get account[C1] and create cloudspace on it
+                                    , should fail to updare account[C1] name. 
+        #. If accesstype admin [ARCXDU], should succeed to get , update [C1] name and  create cloudspace on it. 
```

Comment:
Add 'Delete' too

Reply:
there is no delete in cloudapi , and i need user to be in user group only